### PR TITLE
Create vendor-status.md

### PR DESCRIPTION
**Pull request by jasti**
Provide the current status of all Ad-tech vendors

### CODE REVIEW

File: ads/vendor-status.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+| Ad Server      | Ad Tech     |   First Class Support                   |[Implementation] (https://github.com/ampproject/amphtml/blob/master/ads/adtech.md)               |
+| Ad Server      | Smart Ad Server      |First Class Support                   | [Implementation] (https://github.com/ampproject/amphtml/blob/master/ads/smartadserver.md)              |
+| Ad Server      | YieldMo    |   First Class Support                  |[Implementation] (https://github.com/ampproject/amphtml/blob/master/ads/yieldmo.md)               |
+| Ad Server      | Dot&Ads    |   First Class Support                  |[Implementation] (https://github.com/ampproject/amphtml/blob/master/ads/dotandads.md)               |
+| Video Player     | Brightcove    |   First Class Support                  |[Implementation] (https://github.com/ampproject/amphtml/blob/master/extensions/amp-brightcove/amp-brightcove.md)|
+| Viewability    | MOAT    |   First Class Support                  |Please contact MOAT directly
+| Video Player     | YouTube   |   First Class Support                  |[Implementation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-youtube/amp-youtube.md)|
```
Comment (jmadler) [inline ads/vendor-status.md]: Is YouTube an adtech company?  I'm not certain it belongs here

File: ads/vendor-status.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+| Video Player     | Active View    |   First Class Support                  |Please contact Doubleclick directly|
```
Comment (jmadler) [inline ads/vendor-status.md]: Does Doubleclick stylize themselves that way?  or is it DoubleClick?

File: ads/vendor-status.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+## AdTech Vendor Integration Status
+There are a number of adtech vendors that are actively integrating with AMP. If you don't find your ad tech provider listed below, we encourage you to invite the vendor directly to create first class support in AMP where it makes sense.
+If you are adtech vendor and would like to provide AMP support, please start by reading the ads [integration guide](https://github.com/ampproject/amphtml/blob/master/ads/integration-guide.md). 
+
+This page is updated weekly.
+
+
+
+| Vendor Type        | Vendor Name   | Integration Status | Documentation |
```
Comment (jmadler) [inline ads/vendor-status.md]: I'd move the Vendor Name column to the left as it's the most relevant column

File: ads/vendor-status.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+## AdTech Vendor Integration Status
+There are a number of adtech vendors that are actively integrating with AMP. If you don't find your ad tech provider listed below, we encourage you to invite the vendor directly to create first class support in AMP where it makes sense.
+If you are adtech vendor and would like to provide AMP support, please start by reading the ads [integration guide](https://github.com/ampproject/amphtml/blob/master/ads/integration-guide.md). 
+
+This page is updated weekly.
```
Comment (jmadler) [inline ads/vendor-status.md]: I'd remove this line.  It'll be kept up to date, but providing a specific frequency is unrealistic :)

File: ads/vendor-status.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+## AdTech Vendor Integration Status
+There are a number of adtech vendors that are actively integrating with AMP. If you don't find your ad tech provider listed below, we encourage you to invite the vendor directly to create first class support in AMP where it makes sense.
+If you are adtech vendor and would like to provide AMP support, please start by reading the ads [integration guide](https://github.com/ampproject/amphtml/blob/master/ads/integration-guide.md).
```
Comment (jmadler) [inline ads/vendor-status.md]: Can you add a note in the integration guide that adtech vendors should add themselves here?

**Comment by jasti** [comment]
Will submit revised PR. Closing this one.

**Pull request closed without merging**